dist/emoji-en-US.json

```diff
@@ -17223,6 +17223,12 @@
     "romania",
     "romanian"
   ],
+  "🇨🇶": [
+    "flag_sark",
+    "cq",
+    "flag",
+    "banner"
+  ],
```

Comment:
Most of the flags also have a `nation` and `country` as keywords. I intentionally omitted those keywords for this emoji, because [apparently Sark is not technically a country](https://www.reddit.com/r/AskUK/comments/3qtk28/is_sark_considered_as_a_country/?rdt=45466).